files/etc/bash.bashrc

```diff
@@ -334,6 +334,17 @@ if test -r /etc/profile.d/vte.sh -a ! -k /etc/profile.d/vte.sh; then
   . /etc/profile.d/vte.sh
 fi
 
+if test "$_is_save" = "unset" ; then
+    #
+    # Just in case the user excutes a command with ssh or sudo
+    #
+    if test \( -n "$SSH_CONNECTION" -o -n "$SUDO_COMMAND" \) -a -z "$PROFILEREAD" -a "$noprofile" != true ; then
+	_SOURCED_FOR_SSH=true
```

Comment:
I suggest to add here

                 _is_save="$is"

to reset it later on ...